config/samples/oas/client_certificate.yaml

```
@@ -0,0 +1,51 @@
+# Secret is not created in this manifest.
+# Please store client certificate in k8s TLS secret `tls-cert`.
+
+apiVersion: v1
+data:
+  test_oas.json: |-
+    {
+        "info": {
+          "title": "Petstore",
+          "version": "1.0.0"
+        },
+        "openapi": "3.0.3",
+        "components": {},
+        "paths": {},
+        "x-tyk-api-gateway": {
+          "info": {
+            "name": "Petstore",
+            "state": {
+              "active": true
+            }
+          },
+          "upstream": {
+            "url": "https://petstore.swagger.io/v2"
+          },
+          "server": {
+            "listenPath": {
+              "value": "/petstore/",
+              "strip": true
+            },
+            "clientCertificates":{
+              "enabled": true
+            }
+          }
+        }
+      }
+kind: ConfigMap
+metadata:
+  name: cm
+  namespace: default
+---
+apiVersion: tyk.tyk.io/v1alpha1
+kind: TykOasApiDefinition
+metadata:
+  name: petstore
+spec:
+  tykOAS:
+    configmapRef:
+      name: cm
+      namespace: default
+      keyName: test_oas.json
+  client_certificate_ref: [tls-cert]
```

I guess this should be camelCase as it is defined as camelCase in Go spec.
```suggestion
  clientCertificateRef: [tls-cert]
```